pom.xml

```diff
@@ -0,0 +1,483 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+    http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.openmicroscopy</groupId>
+  <artifactId>ome-codecs</artifactId>
+  <version>6.0.0-SNAPSHOT</version>
```

Comment:
As we are changing the `groupId`/`artifactId` didn't we decide on going to 0.1.0-SNAPSHOT for now?